modules/src/twitter.py

```diff
@@ -101,6 +101,10 @@ def getDesc(req):
     return desc
 
 
+def startwith(char):
```

Comment:
Hey, I overlooked this again, not a blocker, that you have constructed your own method for it but I was talking about the python's built in one, let's use the built one, please :) 
Reference 

- https://docs.python.org/3/library/stdtypes.html
- http://python-reference.readthedocs.io/en/latest/docs/str/startswith.html?highlight=startswith

Reply:
Oh my bad I didn't know this built-in function ;) I amended with an updated version. All my apologizes!